lib/cli.js

```diff
@@ -432,6 +461,14 @@ module.exports = async (argv) => {
 		optionsBase.disableDefaultIgnores = cli.flags.disableDefaultIgnores;
 	}
 
+	if (cli.flags.ext) {
+		optionsBase.extensions = cli.flags.ext
+			.replace(/\./g, '')
```

Comment:
[suggest] I think moving this `replace()` into `standalone.js` is necessary because Node.js API also allows `['.css', '.scss']`, not only `['css', 'scss']`.

Reply:
For example, we can use `replace()` in `filterByExtensions()` like this:

```js
const extList = extensions.map((ext) => ext.replace(/\./g, ''));
```

Reply:
> because Node.js API can also allow `['.css', '.scss']`

That's not allowed.

If you are saying it should also be allowed. I think it should be discussed in #6689.
It makes sense to have the `.` with `--ext` but not really with an array.
In this case I favor discoverability/conventions over consistency.
Let's keep it simple for now and add it later?

<hr />

BTW I think you could be interested in [this RFC](https://github.com/eslint/rfcs/pull/9/files#diff-3553ca3bc6ff6b51efa315c689191a841deb1f6fdb83f9b318ffb2c3a6adf6a8R550).
It kinds of support your argument of a possible conflict between `files` and config `extensions` but it also favors the config over the cli—which is the opposite of what we are currently doing.

Reply:
> Let's keep it simple for now and add it later?

Make sense. Let's keep it as-is. 👍🏼 

> It kinds of support your argument of a possible conflict between files and config extensions but it also favors the config over the cli—which is the opposite of what we are currently doing.

Yeah, we can learn many lessons from ESLint. This [ESLint blog](https://eslint.org/blog/2022/08/new-config-system-part-1/#the-need-for-simplification) emphasizes also **simplicity**. The more we fulfill people's requests, the harder we maintain the codebase.

Although I believe the `--ext` support is convenient for now, we may face difficulties if we extend the config system in the future.

Could you share the links on #3411 and #3860?

Reply:
Now that I have added support for `--ext css,scss`, your request is now more justified.
I consider the PR ready as is but if you want I could add support for `['.css', '.scss']` for consistency sake.

Reply:
> I consider the PR ready as is but if you want I could add support for `['.css', '.scss']` for consistency sake.

Thanks. I think it's optional, but if we can realize this with a few codes, it's desirable.